bigtable/client/mutations.h

```diff
@@ -41,13 +42,27 @@ struct Mutation {
 Mutation SetCell(std::string family, std::string column,
                  std::chrono::milliseconds timestamp, std::string value);
 
+/// Create a mutation to set a cell with numeric value.
+Mutation SetCell(std::string family, std::string column,
+                 std::chrono::milliseconds timestamp,
+                 bigtable::internal::bigendian64_t value);
```

Comment:
If `bigendian64_t` is part of the public interface then we need to move it out of `bigtable::internal`.  What do you think of defining a standalone function:

```C++
namespace bigtable { inline namespace BIGTABLE_CLIENT_NS {
  std::string as_bigendian(std::int64_t x);
}
}
```

Then the application can just say:

```C++
auto mut = bigtable::SetCell(..., bigtable::as_bigendian(1234));
```

That saves us some overloads for `SetCell()`.